lib/utils/messages.js

```diff
@@ -461,11 +475,43 @@ module.exports = function () {
       var message       = action.options.message;
       var colorPrinter  = action.options.color;
 
-      filesChanged.forEach(function (file) {
-        output.push(colorPrinter([prefix, file, message].join(' ')));
-      });
+      if (typeof filesChanged === 'string') {
+        output.push(colorPrinter([prefix, filesChanged, message].join(' ')));
+      } else {
+        filesChanged.forEach(function (file) {
+          output.push(colorPrinter([prefix, file, message].join(' ')));
+        });
+      }
     });
 
     return output.join('\n');
   }
+
+  function howToSolveConflicts(conflicts) {
+    var output = 'To resolve run:\n';
+    output += '- ' + TOOL_NAME + ' pull (overwrites local changes)\n';
+    if (conflicts.rootRamlDeleted) {
+      output += 'or\n';
+      output += '- create ' + conflicts.rootRamlDeleted;
```

Comment:
oh, I got it now, it should be something more descriptive like:
'create a new file and name it: ' + bla